prog/weave-kube/launch.sh

```diff
@@ -41,7 +44,11 @@ fi
 
 # Need to create bridge before running weaver so we can use the peer address
 # (because of https://github.com/weaveworks/weave/issues/2480)
-/home/weave/weave --local create-bridge --force --expect-npc
+if [ "${EXPECT_NPC}" = "0" ]; then
+    /home/weave/weave --local create-bridge --force
+else
+    /home/weave/weave --local create-bridge --force --expect-npc
```

Comment:
Stylistically would prefer not to repeat the elements that are the same, but to create another variable and populate it either with `""` or `"--expect-npc"`.  See how `NICKNAME_ARG` is done, for instance.